eng/scripts/SetTestPipelineVersion.ps1

```diff
@@ -0,0 +1,28 @@
+# Overides the project file and CHANGELOG.md for the template project using the next publishable version
+# This is to help with testing the release pipeline.
+. "${PSScriptRoot}\..\common\scripts\common.ps1"
+$latestTags = git tag -l "Azure.Template*beta*"
```

Comment:
I'd suggest just `Azure.Template_*`.

Reply:
I'll try that. It should work as long as the sorting in semvar can handle preview and beta.